recipes/cath-tools/build.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+set -x -e
+
+export INCLUDE_PATH="${PREFIX}/include"
+export LIBRARY_PATH="${PREFIX}/lib"
+export LD_LIBRARY_PATH="${PREFIX}/lib"
+export CFLAGS="-I$PREFIX/include"
+export CPATH="${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+export CXXFLAGS="-I${PREFIX}/include"
+
+
+#list programs
+CATH_PROGRAMS="build-test cath-assign-domains cath-cluster cath-map-clusters cath-refine-align cath-resolve-hits cath-score-align cath-ssap cath-superpose"
+
+#compile
+DCMAKE=""
+if [[ ${HOST} =~ .*darwin.* ]]; then
+	MACOSX_DEPLOYMENT_TARGET=10.9
+	DCMAKE+=" -DCMAKE_THREAD_LIBS_INIT=-lpthread -DCMAKE_HAVE_THREADS_LIBRARY=1"
+	DCMAKE+=" -DCMAKE_USE_PTHREADS_INIT=1"
+	export CXXFLAGS="${CXXFLAGS} -fvisibility=hidden -fvisibility-inlines-hidden"
```

Comment:
I am guessing that you don’t need to modify CXXFLAGS here either. I believe that the CXXFLAGS set by Conda already include those flags.

Reply:
Without those extra CXXFLAGS I get lot of warning of this type: 
```
 (ERR) ld: warning: direct access in function 'boost::filesystem::detail::canonical(boost::filesystem::path const&, boost::filesystem::path const&, boost::system::error_code*)' from file '//anaconda3/envs/bioconda/conda-bld/cath-tools_1587726492986/_build_env/lib/libboost_filesystem.a(operations.o)' to global weak symbol 'boost::system::detail::cat_holder<void>::generic_category_instance' from file 'libct_common.a(program_exception_wrapper.cpp.o)' means the weak symbol cannot be overridden at runtime. This was likely caused by different translation units being compiled with different visibility settings.
```